pkg/cmd/dev.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	"github.com/snyk/go-application-framework/pkg/app"
+	"github.com/snyk/go-application-framework/pkg/configuration"
+	"github.com/snyk/go-application-framework/pkg/workflow"
+	"github.com/spf13/cobra"
+)
+
+// Dev is a helper utility for extension authors to define a lightweight CLI to
+// test their extensions outside of the main Snyk CLI.
+func Dev(initializers ...workflow.ExtensionInit) (*cobra.Command, error) {
+	// Initialize the engine with the given workflows
+	logger := log.New(os.Stderr, "", 0)
+	engine := app.CreateAppEngineWithLogger(logger)
+	engine.SetConfiguration(configuration.New())
+	for _, i := range initializers {
+		engine.AddExtensionInitializer(i)
```

Comment:
Could this be also an option that can be passed to `CreateAppEngineWithOpts`? E.g. `WithExtensionInitializers`?

Reply:
Thanks! Addressed in d569237